src/components/Employee.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { dateFormat } from '../utils/dateFormatter';
+
+export default class Employee extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { employee } = this.props;
+    return (
```

Comment:
Here you may do employee destructuring with default values and not use below employee.username, employee.first_name ...

Reply:
I don't get it.

Reply:
const { employee = {} } = this.props;
    const {username, first_name, last_name, email, created_at} = employee